Source/swiftlint/Extensions/Configuration+CommandLine.swift

```diff
@@ -66,20 +66,16 @@ extension Configuration {
         for file in files {
             // Files whose configuration specifies they should be excluded will be skipped
             let fileConfiguration = configuration(for: file)
-            let excludedPaths = fileConfiguration.excluded
-                .map { (fileConfiguration.rootPath ?? "").bridge().appendingPathComponent($0) }
```

Comment:
By folding this `map` into the `contains` check below, we avoid iterating and appending paths as soon as one would have been "skippable".